review-me/review-me.js

```diff
@@ -0,0 +1,26 @@
+// hello.js
+
+var message = "Hello, world!"
+console.log(MESSAGE);
+
+function printmessage(msg) {
+    if(msg != null){
+        console.log(msg)
+    }
+    else{
+        console.log("Nothing to print")
+    }
+}
+
+printmessage(message)
+
+eval("console.log('Eval should not be used!')")
+
+let unused = 123
+
+function DoStuff()
+{
+console.log   (     "Stuff" );
```

Comment:
23  :  excessive  spacing  in  ' console .log     (       " Stuff "  ); ',  code  style  issue